src/NuGet.Core/NuGet.Protocol.Core.Types/Resources/IPackageSearchResource.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace NuGet.Protocol.Core.Types
+{
+    public interface IPackageSearchResource : INuGetResource
```

Comment:
Is there a reason why this is an interface for this when the rest are abstract classes?